src/multiselect_controls.js

```diff
@@ -506,7 +506,6 @@ export class MultiselectControls {
     // Revert Gesture.setStartBlock to original settings
     // when disabling multiselect mode.
     Blockly.Gesture.prototype.handleBlockStart = this.origHandleBlockStart;
-    Blockly.Gesture.prototype.setStartBlock = this.origSetStartBlock;
```

Comment:
Ah, looks like you only modified the `handleBlockStart`, the comment here confuses me, please correct that